app/frontend/src/i18n/formatters.tsx

```diff
@@ -0,0 +1,76 @@
+export default {
+    lowercase,
+    uppercase,
+    datetime,
+    number,
+    currency
+} as const;
+
+export function lowercase(value: string) {
```

Comment:
Are these functions all expected by i18next? I can't find any direct use of them in the repository. Can you add a comment/link so that people don't accidentally delete them, thinking they're unused?

Reply:
Hm, I'm reading https://www.i18next.com/translation-function/formatting but it kind of looks like these are the legacy formatters approach?

Reply:
Thank you very much for your feedback. I have changed the code to use the built-in formatting functions instead of the legacy formatters, so this file is no longer needed.